docs/docs-beta/docs/guides/external-systems/non-python.md

```diff
@@ -1,4 +1,141 @@
 ---
 title: "Running non-Python languages"
 sidebar_position: 60
----
\ No newline at end of file
+---
+
+Dagster is written in Python, but that doesn't mean it's limited to running Python to materialize assets. This guide covers how to run some JavaScript with Dagster using Pipes.
+
+# Define your JavaScript function
+
+Here's a contrived example of some JavaScript code to run using Dagster Pipes. `train_model` load some dataset and train a sequential model using tensorflow.
+
+```javascript
```

Comment:
In our Dagster asset we reference this file as:
``` 
command=["node", "../tensorflow/main.js"],
```
Can we include the file header for this code block?

Reply:
This file should be extracted out to a `.js` file in code_examples. That way it can be auto-formatted.